test/akamai-test-srv/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"sync"
+
+	"github.com/letsencrypt/boulder/akamai"
+	"github.com/letsencrypt/boulder/cmd"
+)
+
+func main() {
+	listenAddr := flag.String("listen", "localhost:6789", "Address to listen on")
+	secret := flag.String("secret", "", "Akamai client secret")
+	flag.Parse()
+
+	// v2
+	v2Purges := [][]string{}
+	v3Purges := [][]string{}
+	mu := sync.Mutex{}
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think it would be nicer to see this broken up into separate handler functions registered on a mux. Right now its a pretty long function and has to do its own path checking.